README.md

```diff
@@ -1,3 +1,4 @@
+# CFPB development team
 
 > A guide for the development team at the [CFPB](http://cfpb.github.io/).
```

Comment:
While we're here, let's pull this out of the blockquote styling and make this link HTTPS.